smqtk_image_io/gdal_io.py

```diff
@@ -8,8 +8,8 @@
 import six
 from six.moves import range
 
-from smqtk.algorithms import ImageReader
-from smqtk.utils.image import crop_in_bounds
+from smqtk_image_io import ImageReader
```

Comment:
Sorry I dont think i understand exactly what you mean here.
so instead of `from smqtk_image_io import ImageReader` this should just be `import ImageReader` ?

Reply:
I meant in that, since `ImageReader` is a type defined in this package, this module may want to use `from smqtk_image_io.interfaces.image_reader import ImageReader` as it's import. This is a little more verbose, but it will directly address the module in which the type is implemented. I only say this because I've had issues in the past related to testing where a direct import was the solution as opposed to the convenience import. I can ease off on this if you have strong opinions otherwise.